Modules/Core/SpatialObjects/include/itkMetaArrowConverter.hxx

```diff
@@ -53,17 +55,19 @@ MetaArrowConverter<VDimension>::MetaObjectToSpatialObject(const MetaObjectType *
     positionInObjectSpace[i] = metaPosition[i];
     directionInObjectSpace[i] = metaDirection[i];
   }
-  arrowSO->SetPositionInObjectSpace(positionInObjectSpace);
+  if (mo->APIVersion() == 1)
+  {
+    // if not using new API, then the position is already stored as Offset.
+    // Really not certain how to handle the old case since it produced a bug...with
+    //    position being stored in the Offset and the Position!  But then the WorldPosition
+    //    would compose offset with position, which would be wrong...   It really never
+    //    worked correctly...
+    arrowSO->SetPositionInObjectSpace(positionInObjectSpace);
+  }
+
+  std::cout << directionInObjectSpace << std::endl;
```

Comment:
rm debug code

Reply:
Doh!

Reply:
Thanks @aylward Can you please also squash this one ("COMP: Removing debugging cout ") with the other commits?

FYI, it's common practice to process review comments and CI failures on a pull request by "amending" the commit(s) of the pull request, and then _force-pushing_ your adjustments (`git review-push --force`) rather than by adding extra commits.